src/GitHub.Exports/Helpers/AssemblyResolver.cs

```diff
@@ -51,7 +52,7 @@ static Assembly LoadAssemblyFromRunDir(object sender, ResolveEventArgs e)
             {
                 var requestedName = e.Name.TrimSuffix(".dll", StringComparison.OrdinalIgnoreCase);
                 var name = new AssemblyName(requestedName).Name;
-                if (!ourAssemblies.Contains(name))
+                if (!ourAssemblies.Contains(name, new LambdaComparer<string>((lhs, rhs) => String.Compare(lhs, rhs, StringComparison.OrdinalIgnoreCase))))
```

Comment:
Oh duh. My brain today...

andreia
github.com/shana
@sh4na <http://twitter.com/sh4na>


On Mon, Aug 29, 2016 at 6:16 PM, Phil Haack <notifications@github.com>
wrote:

> In src/GitHub.Exports/Helpers/AssemblyResolver.cs
> <https://github.com/github/VisualStudio/pull/520#discussion_r76636504>:
>
> > @@ -51,7 +52,7 @@ static Assembly LoadAssemblyFromRunDir(object sender, ResolveEventArgs e)
> >              {
> >                  var requestedName = e.Name.TrimSuffix(".dll", StringComparison.OrdinalIgnoreCase);
> >                  var name = new AssemblyName(requestedName).Name;
> > -                if (!ourAssemblies.Contains(name))
> > +                if (!ourAssemblies.Contains(name, new LambdaComparer<string>((lhs, rhs) => String.Compare(lhs, rhs, StringComparison.OrdinalIgnoreCase))))
>
> I think you could just pass in StringComparer.OrdinalIgnoreCase here
> rather than the LambdaComparer. 😄
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/VisualStudio/pull/520/files/4e1632ad6e348c594b2d7a0b7bc5cd507843a3a6..c1fc0c8a709c99181e899316d8866ebee65c5c3c#r76636504>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAS7ebWSD2yLEsGu-J-EIfqyTRJrPHsCks5qkwXlgaJpZM4Jt_kb>
> .
>